testserver/testserver.go

```diff
@@ -118,6 +122,17 @@ func NewTestServer() (*TestServer, error) {
 		return nil, fmt.Errorf("could not create logs directory: %s: %s", logDir, err)
 	}
 
+	options := url.Values{
+		"host": []string{baseDir},
```

Comment:
yeah, it's really gross. That's the reason I've kept the `--socket` flag hidden for now. If there's real demand for it, we can make it more publicly-visible and more in line with postgres, but for now, this is much simpler to use, especially for the dynamic case.